ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/FieldInfo.java

```diff
@@ -31,22 +31,26 @@
 
   public enum FieldType {
     SYSTEM, // To be removed in the future. 0.9 saw this value no longer used.
```

Comment:
This can be removed, right? I don't see being used either.

Reply:
The purpose of keeping this was so that pre-0.9 servers could still work with newer CLIs. I'm not sure how many people are still using 0.9, but I think this is easy enough to just keep anyway. This does raise a good point though - the changes in this PR would make HEADERS require the newest CLI. Are we okay with that?